docs/static/monitoring/monitoring-prereq-setup-es.asciidoc

```diff
@@ -1,5 +1,5 @@
 //[[set-up-monitoring]]
 //=== Set up {es} monitoring (stack monitoring only)
 
-Set up {ref}/monitoring-overview.html[{es} monitoring].
+With *Stack Monitoring* it is recommended to set up {ref}/monitoring-overview.html[{es} monitoring] together. Otherwise, {ls} becomes a Standalone Cluster.
```

Comment:
review note: not sure if it is okay to mention standalone cluster but if user doesn't configure the ES monitoring, LS becomes an orphan.